config.json

```diff
@@ -315,7 +315,16 @@
       "core": true,
       "difficulty": 4,
       "slug": "saddle-points",
-      "topics": null,
+      "topics": [
+        "algorithms",
+        "arrays",
+        "conditionals",
+        "integers",
+        "loops",
+        "mathematics",
+        "matrices",
+        "variables"
```

Comment:
Great suggestions! :) The only ones I'm not sure about are `algorithms` and `variables`. They do apply to this exercise but they also apply to every other exercise. Why do you think they specifically relate to this one?

I would also add `lists` and `sets` since you have to use them to pass the tests :)

Reply:
it's done